hips/tiles/tests/test_surveys.py

```diff
@@ -85,8 +96,6 @@ def test_fetch(self):
         surveys = HipsSurveyPropertiesList.fetch()
         assert len(surveys.data) > 3
 
-        # TODO: look up survey by name here
-        # Otherwise this will break when the list changes
-        survey = surveys.data[0]
+        survey = HipsSurveyProperties.from_name('CDS/P/2MASS/H')
```

Comment:
@cdeil I'm not sure what you mean here. Do you mean to say create a simple class method `from_name` in addition to the `classmethod`?